src/app/routes/home/getInitialData/index.js

```diff
@@ -1,4 +1,8 @@
 import pipe from 'ramda/src/pipe';
+import pathOr from 'ramda/src/pathOr';
+import { getRadioScheduleEndpoint } from '#lib/utilities/getRadioSchedulesUrls';
+import serviceConfigs from '#server/utilities/serviceConfigs'; // or we could just have an array of services.
```

Comment:
This massssssively increases the bundle size (as the file warns), so will need to think of another way of getting this config (the most lofi being defining an array in here)